### DISCUSSION TITLE
Boss pode funcionar com SourceForge?

### DISCUSSION THREAD

**Discussion by marcelinoap** [HashLoad/boss#168] [Q&A] [answered]
Tenho um projeto que possui dependências de componentes alocados no SourceForge. Tentei fazer o boss install "link sourceforge" mas sempre dá "Error ... repository not found" 

O Boss não é compatível com ele?

**Comment by viniciussanchez** [comment, marked as answer]
Bom dia... apenas com repositórios Git.
